src/App.js

```diff
@@ -1,25 +1,40 @@
 import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Root from './pages/Root';
+import Beers from './pages/Beers';
+import RandomBeer from './pages/RandomBeer';
```

Comment:
this dont even  run since you dont have this pages created also you dont install antd or react-router-dom?